config/config.go

```diff
@@ -145,6 +145,9 @@ type BaseConfig struct {
 	// Output level for logging
 	LogLevel string `mapstructure:"log_level"`
 
+	//Structured Logging as JSON or colored text
+	StructuredLog bool `mapstructure:"structured_log"`
```

Comment:
I'd call it `LogFormat string` ("plain", "json"). Log lines we output to stdout also have structure, so it's not fair to call them unstructured.